CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Add support for specifying additional extensions to load ([#547], [#563])
+- Support Druid `30.0.0` ([#583])
```

Comment:
We usually mention the removed support (`27.0.0`) as well.

Reply:
[`cb75be6` (#583)](https://github.com/stackabletech/druid-operator/pull/583/commits/cb75be69a08bae32b2a4bdd96a444fbd7b8da3aa)